feature_apps/src/main/java/cm/aptoide/pt/feature_apps/presentation/AppGraphicView.kt

```diff
@@ -55,39 +59,63 @@ fun AppGraphicView(
           .height(136.dp)
           .clip(RoundedCornerShape(16.dp))
       )
-      if (bonusBanner) {
-        Text(
-          text = "up to\n20%\nBONUS",
-          textAlign = TextAlign.Center,
-          fontSize = 12.sp,
-          color = MaterialTheme.colors.primary,
+      bonusAppcModel?.bonusPercentage?.let {
+        Box(
           modifier = Modifier
-            .background(
-              Color.White,
-              RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp, bottomEnd = 16.dp)
-            )
             .size(64.dp)
-            .padding(8.dp)
-        )
+            .offset(x = (-0.1).dp, y = (-0.1).dp),
+          contentAlignment = Alignment.Center
+        ) {
+          Image(
+            imageVector = AppTheme.icons.BonusBackground,
+            contentDescription = "BonusBackground",
+            modifier = Modifier.fillMaxSize()
+          )
+          Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy((-3).dp)
+          ) {
+            BonusText(text = "up to", fontWeight = FontWeight.Bold)
```

Comment:
hardcoded text

Reply:
The translations issue is a big one and better to resolve in another ticket